### PR TITLE
Add transaction history

### PR DESCRIPTION
**Pull request by fel-w**
# Description

Add template for transaction history of previous payments and their bill details.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Trello Ticket ID

https://trello.com/c/pjGHwfTy

## How Can This Been Tested?
Checkout the branch and go to the billing page to view the transaction history as well

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

# Screenshots

image

image

### CODE REVIEW

File: src/components/ProjectBillingPage/ProjectBillingPage.module.css [hunk 1]
```diff
@@ -83,7 +83,6 @@
     padding:  0 1rem;
     font-weight: 800;
 }
-
 .MonthSummary{
     display: grid;
     grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1.5fr;
```
Comment (neelxie) [inline src/components/ProjectBillingPage/ProjectBillingPage.module.css:88]: Remove this line as it has no effect.

File: src/components/ProjectBillingPage/ProjectBillingPage.module.css [hunk 2]
```diff
@@ -181,18 +174,90 @@
...
-
+.TransactionHistoryWrapper {
+  padding-top: 1.5rem;
+}
+.TransactionHistoryContainer{
+  border: 1px solid #cccccc;
+  background-color: white;
+}
+.TransactionHistoryHeading {
+  padding: 1rem;
```
Comment (neelxie) [inline src/components/ProjectBillingPage/ProjectBillingPage.module.css:185]: duplication

**Review by neelxie — APPROVED** [review]
LGTM

**Review by rhodinemma — APPROVED** [review]
Looks good

**Review by khalifan-kfan — APPROVED** [review]

**Pull request merged**